Source/Applications/SystemCenterNotification/Scripts/TSX/CellCarrier/ByCellCarrier.tsx

```diff
@@ -54,8 +54,9 @@ const ByCellCarrier = (props: IProps) => {
     const sortField = useAppSelector(CellCarrierSlice.SortField);
     const asc = useAppSelector(CellCarrierSlice.Ascending);
     const filters = useAppSelector(CellCarrierSlice.SearchFilters);
-    const [showModal, setShowModal] = React.useState<'New'|'Edit'|'Hide'>('Hide');
-    const [carrier, setCarrier] = React.useState<ICellCarrier>({ Name: '', Transform: '', ID: 0 })
+    const [showModal, setShowModal] = React.useState<'New' | 'Edit' | 'Hide'>('Hide');
+    const emptyCarrier = { Name: '', Transform: '', ID: 0 };
```

Comment:
if this const in not mutated, it should be outside of the component so it doesn't get reassigned every render